.drone.yml

```diff
@@ -129,13 +132,46 @@ pipeline:
       event: deployment
       environment: qa
 
-  deploy-to-prod:
+  deploy-to-wcs-qa:
     image: quay.io/ukhomeofficedigital/kd:v1.16.0
     environment:
+      - KUBE_NAMESPACE=wcs-qa
+      - ENVIRONMENT=qa
```

Comment:
You don't need this; you could use [$DRONE_DEPLOY_TO](https://0-8-0.docs.drone.io/environment-reference/).

Reply:
Agreed, but as I said in the description, QA and PROD will be dealt with in the next wave of changes, as kube_namespace is being removed completely.